packages/sdk/src/utils/index.ts

```diff
@@ -97,57 +91,9 @@ export function tapTweakHash(pubKey: Buffer, h: Buffer | undefined): Buffer {
   return bitcoin.crypto.taggedHash("TapTweak", Buffer.concat(h ? [pubKey, h] : [pubKey]))
 }
 
-export function calculateTxFee({
-  totalInputs,
-  totalOutputs,
-  satsPerByte,
-  type,
-  additional: { witnessScripts = [] } = {}
-}: CalculateTxFeeOptions): number {
-  const txWeight = calculateTxVirtualSize({ totalInputs, totalOutputs, type, additional: { witnessScripts } })
-  return txWeight * satsPerByte
-}
-
-export function calculateTxVirtualSize({
-  totalInputs,
-  totalOutputs,
-  type,
-  additional: { witnessScripts = [] } = {}
-}: CalculateTxVirtualSizeOptions) {
-  const baseWeight = getInputOutputBaseSizeByType(type)
-
-  const inputVBytes = baseWeight.input * totalInputs
-  const outputVBytes = baseWeight.output * totalOutputs
-  const baseVBytes = inputVBytes + outputVBytes + baseWeight.txHeader
-  const additionalVBytes = witnessScripts.reduce((acc, script) => (acc += script.byteLength), 0) || 0
-
-  const weight = 3 * baseVBytes + (baseVBytes + additionalVBytes)
-  const vSize = Math.ceil(weight / 4)
-
-  return vSize
-}
-
-export function getInputOutputBaseSizeByType(type: AddressFormats) {
-  switch (type) {
-    case "taproot":
-      return { input: 57.5, output: 43, txHeader: 10.5 }
-
-    case "segwit":
-      return { input: 68, output: 31, txHeader: 10.5 }
-
-    case "nested-segwit":
-      return { input: 68, output: 32, txHeader: 10.5 }
-
-    case "legacy":
-      return { input: 147.5, output: 34, txHeader: 10.5 }
-
-    default:
-      throw new Error("Invalid type")
-  }
-}
-
 export const isObject = (o: any) => o?.constructor === Object
 export const isString = (s: any) => s instanceof String || typeof s === "string"
+export const isFloat = (n: number) => Number(n) === n && n % 1 !== 0
```

Comment:
hmm not sure if this is a good way to determine is a number is float. 
For instance, a big enough float number would fail this check
ex `isFloat(9.0000000000000001)`

Reply:
Good catch! fwiw, the outcome in your example is expected because numbers in Javascript loses precision. Improvised the handling; [ref](https://github.com/sadoprotocol/ordit-sdk/pull/62/commits/336e5a1fd27268eac917b75c719eda2e04bb8048)